Project-Aurora/Project-Aurora/Settings/KeyboardLayoutManager.cs

```diff
@@ -750,6 +750,28 @@ public void LoadBrand(PreferredKeyboard keyboard_preference = PreferredKeyboard.
                 layoutConfigPath = Path.Combine(layoutsPath, "drevo_blademaster.json");
             else if (keyboard_preference == PreferredKeyboard.Wooting_Two)
                 layoutConfigPath = Path.Combine(layoutsPath, "wooting_two.json");
+            else if (keyboard_preference == PreferredKeyboard.Keyboard101)
+                layoutConfigPath = Path.Combine(layoutsPath, "keyboard101.json");
```

Comment:
Can't you make these json names more descriptive?

Reply:
OK , I am going to modify to our keyboard type name.